typescript/cli/examples/submit/strategy/gnosis-chain-tx-builder-strategy.yaml

```diff
@@ -0,0 +1,7 @@
+basesepolia:
+  submitter:
+    chain: 'basesepolia'
```

Comment:
can we remove this chain field from here?

Reply:
not yet. this is because we rely in `submit()` which only accepts 1 parameter. We'd have to add chain into it so it knows where it's submitting, which requires an update to the abstract.